language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -40,18 +54,32 @@ optionsParser = Options
         <> metavar "DIR"
         <> help "Output directory for the generated TypeScript files"
         )
+    <*> optional (strOption
+        (  long "daml-types-version"
+        <> help "The version of '@daml/types' to depend on (warning : not a user-configuration parameter; for SDK internal use)"
+        ))
+    <*> optional (strOption
+        (  long "package-version"
+        <> help "The version string to use for output packages (warning : not a user-configuration parameter; for SDK internal use)"
+        ))
+    <*> optional (strOption
+        (  short 'p'
+        <> metavar "PACKAGE-JSON"
+        <> help "A 'package.json' of workspaces"
```

Comment:
When would I want to override this?

Reply:
Oh I see, this is the path where the `package.json` file will be written to. The `help` text is confusing. I thought I pass in the `package.json` myself.

Reply:
i'll do something with the help text